### PR TITLE
docs(tenant-selector): rewrite docs in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-564.gearboxjs.preview.cogniteapp.com

File: src/components/TenantSelector/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    lightGrey: 'white',
+    buttonDisabledColor: '#DDD',
+    lightShadow: 'rgba(0, 0, 0, 0.15) 10px 10px 8px 8px',
+  },
+};
+```
+
+<Preview>
+  <Story name="With Theme" >
+    <ThemeProvider theme={themeExample}><TenantSelector title="Styled App" onTenantSelected={onTenantSelected} /></ThemeProvider>
```
Comment (mrtraser) [inline src/components/TenantSelector/stories/basic.story.mdx]: alignment

Comment (amevck) [reply src/components/TenantSelector/stories/basic.story.mdx]: There is a wrapper component. So better showing in the code snippet we need to keep this inline like this.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-564.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-564.gearboxjs.preview.cogniteapp.com

**Pull request merged**